grudge/geometry/metrics.py

```diff
@@ -114,7 +114,11 @@ def forward_metric_nth_derivative(
     if dd is None:
         dd = DD_VOLUME
 
-    inner_dd = dd.with_discr_tag(DISCR_TAG_BASE)
+    if dd.discretization_tag is not DISCR_TAG_BASE:
+        raise ValueError(
+            "The DOF descriptor associated with computing metric derivatives "
+            "must have `DISCR_TAG_BASE` as a discretization tag."
```

Comment:
Why is this the case? I could see a rationale for asking for metric derivaties on non-base discretizations. This seems like a feature removal whose rationale I don't fully understand. Maybe add a comment/improve the error message to explain?

Reply:
From https://github.com/inducer/grudge/pull/194#issuecomment-1004189393: the reason for this change was that I moved the interpolation step out of this function and into the relevant routines in `area_element` and similar (you already commented on those parts of the code below).

Reply:
https://github.com/inducer/grudge/pull/194/commits/a7bcf7d0c41f813562d89d21df66108bb8783b03